src/main/java/com/movie/movie/controller/MovieController.java

```diff
@@ -30,10 +30,24 @@ public List<Movie> getMovies(@RequestParam(required = false) String titleStartsW
     }
 
     @PostMapping("/movies")
-    public ResponseEntity<MoveRegisterResponse> insert(@RequestBody MoveRegisterRequest MoveRegisterRequest, UriComponentsBuilder uriBuilder) {
-        Movie movie = movieService.insert(MoveRegisterRequest.getTitle(), MoveRegisterRequest.getReleaseYear());
+    public ResponseEntity<MovieRegisterResponse> insert(@RequestBody MovieRegisterRequest MovieRegisterRequest, UriComponentsBuilder uriBuilder) {
+        Movie movie = movieService.insert(MovieRegisterRequest.getTitle(), MovieRegisterRequest.getReleaseYear());
         URI location = uriBuilder.path("/movies/{id}").buildAndExpand(movie.getId()).toUri();
-        MoveRegisterResponse body = new MoveRegisterResponse("movie registered successfully");
+        MovieRegisterResponse body = new MovieRegisterResponse("movie registered successfully");
         return ResponseEntity.created(location).body(body);
     }
+
+    @PatchMapping("/movies/{id}")
+    public ResponseEntity<MovieRegisterResponse> update(@PathVariable Integer id, @RequestBody MovieRegisterRequest MovieRegisterRequest) {
```

Comment:
Registerは間違っているので直したいですね。
```suggestion
    public ResponseEntity<MovieUpdateResponse> update(@PathVariable Integer id, @RequestBody MovieRegisterRequest MovieRegisterRequest) {
```

Reply:
「MovieUpdateRequest/Response」と「MovieDeleteResponse」クラスを追加して対応しました。
4afbc7e167b4d958902571c466d8b2e1d10021d5